portal/client/src/pages/StudyList/index.js

```diff
@@ -0,0 +1,115 @@
+import React, { useState } from 'react'
+import { Link } from 'react-router-dom'
+
+import { Card, Level, Box, Button, ButtonGroup, Checkbox, TextBox, Icon, Columns, Table } from '../../components'
+
+import PageContainer from '../../components/composite/PageLayout'
+
+const StudyLists = () => {
+    const [isLoading, setLoading] = useState(false)
+    const [isSubmitting, setIsSubmitting] = useState(false)
+    const data = []
+    const columns = []
+
+    return (
+        <PageContainer
+            hasNoHeader
+            isLoading={isLoading}
+            title={'View study list'}
+            subtitle={'All the study records'}
+            iconClassName="fa-list-alt"
+            className="client-list-container"
+        >
+        <section className="notification is-primary is-small top-banner">
+            <Level>
+            <Level.Left>
+                <div>
+                    <div className="is-size-5 has-text-weight-bold has-text-white">
+                        {'qStudy'}
+                    </div>
+                </div>
+            </Level.Left>
+            <Level.Right>                
+                <div>
+                    <Link to="/instructions" className="button mt-2 mr-2 is-block">
+                        <Icon iConClass="fa-info-circle" />
+                        <span>{'Instructions'}</span>
+                    </Link>
+                </div>
+                <div>
+                    <Link to="/login" className="button mt-2 is-block">                        
+                        <Icon iConClass="fa-sign-out-alt" />
+                        <span>{'Logout'}</span>
+                    </Link>
+                </div>
+            </Level.Right>
+            </Level>
+        </section>
+        <div className="is-size-5 has-text-black-bis mb-5 ml-4">
+            {'Hello, User'}
+        </div>
+        <div className="ml-4 mr-4">
+            <Columns>
```

Comment:
```suggestion
            <Columns className="ml-4 mr-4">
```

Lets add the `className` props to the `Columns` component and use the `className` for `div` at line `51` in here